infra/terraform/test-org/ci-triggers/triggers.tf

```diff
@@ -67,11 +69,166 @@ resource "google_cloudbuild_trigger" "tf_validator" {
   substitutions = {
     _TERRAFORM_VERSION = "0.12.31"
     _TEST_PROJECT      = local.tf_validator_project_id
+    _TEST_FOLDER       = local.tf_validator_folder_id
+    _TEST_ANCESTRY     = local.tf_validator_ancestry
+    _TEST_ORG          = local.org_id
+  }
+
+  filename = ".ci/cloudbuild-tests-integration.yaml"
+}
+
+resource "google_cloudbuild_trigger" "tf_validator_main_integration_tests_tf_12" {
+  name = "tf-validator-main-integration-tests-tf-12"
+  description = "Main branch integration tests for terraform-validator with terraform 12. Managed by Terraform https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/tf-validator/project.tf"
+
+  provider    = google-beta
+  project     = local.project_id
+  github {
+    owner = "GoogleCloudPlatform"
+    name  = "terraform-validator"
+    push {
+      branch = "^main$"
+    }
+  }
+  substitutions = {
+    _TERRAFORM_VERSION = "0.12.31"
+    _TEST_PROJECT      = local.tf_validator_project_id
+    _TEST_FOLDER       = local.tf_validator_folder_id
+    _TEST_ANCESTRY     = local.tf_validator_ancestry
+    _TEST_ORG          = local.org_id
+  }
+
+  filename = ".ci/cloudbuild-tests-integration.yaml"
+}
+
+resource "google_cloudbuild_trigger" "tf_validator_pull_integration_tests_tf_13" {
+  name = "tf-validator-pull-integration-tests-tf-13"
+  description = "Pull request integration tests for terraform-validator with terraform 13. Managed by Terraform https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/tf-validator/project.tf"
+
+  provider    = google-beta
+  project     = local.project_id
+  github {
+    owner = "GoogleCloudPlatform"
+    name  = "terraform-validator"
+    pull_request {
+      branch = ".*"
+    }
+  }
+  substitutions = {
+    _TERRAFORM_VERSION = "0.13.7"
+    _TEST_PROJECT      = local.tf_validator_project_id
+    _TEST_FOLDER       = local.tf_validator_folder_id
+    _TEST_ANCESTRY     = local.tf_validator_ancestry
+    _TEST_ORG          = local.org_id
+  }
+
+  filename = ".ci/cloudbuild-tests-integration.yaml"
+}
+
+resource "google_cloudbuild_trigger" "tf_validator_main_integration_tests_tf_13" {
```

Comment:
nit: Looks like this is the same content as above except version- can we for_each though set of `_TERRAFORM_VERSION`. This will also let us make future version edits in one place.

Reply:
Made this change in [f90baca](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/1143/commits/f90bacac33e3e690cffb306868a852b96705dc00)